fendermint/testing/graph-test/Makefile.toml

```diff
@@ -37,17 +38,62 @@ cargo run -p fendermint_eth_api --release --example greeter -- \
 
 [tasks.graph-setup]
 script = """
-echo "TODO: configure the graph"
+set -x
+echo "Start configuring subgraph..."
+echo "Found deployed contract:"
 cat ${TEST_DATA_DIR}/greeter.json
-echo "TODO: docker-compose up the graph"
+echo "\n"
+
+cd ${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}/fendermint
+cp -r testing/graph-test/subgraph testing/graph-test/test-data
+mkdir testing/graph-test/test-data/subgraph/config
+mkdir testing/graph-test/test-data/subgraph/abis
+cp testing/graph-test/test-data/greeter.json testing/graph-test/test-data/subgraph/config/testnet.json
+cp testing/contracts/Greeter.abi testing/graph-test/test-data/subgraph/abis/IGreeter.json
+cd testing/graph-test/test-data/subgraph
+
+npm install
```

Comment:
Maybe the clearing out of anything installed into the local `node` dependencies directory could be added to the  `make clean` command in `fendermint/Makefile`.

Reply:
Yeah now it should be cleaned.